### PR TITLE
add LessEqualPartly function

### PR DESCRIPTION
**Pull request by Thor-wl**
ref： https://github.com/volcano-sh/volcano/issues/1525
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

File: pkg/scheduler/api/resource_info.go [hunk 1]
```diff
@@ -399,6 +399,38 @@ func (r *Resource) LessInSomeDimension(rr *Resource) bool {
 	return false
 }
 
+// LessEqualInSomeDimension returns true if there exists any dimension whose resource amount in r is less than or equal with that in rr.
+// Otherwise returns false.
+// @param defaultValue "default value for resource dimension not defined in ScalarResources. Its value can only be one of 'Zero' and 'Infinity'"
+func (r *Resource) LessEqualInSomeDimension(rr *Resource, defaultValue DimensionDefaultValue) bool {
```
Comment (k82cn) [inline pkg/scheduler/api/resource_info.go:405]: prefer to rename to `LessEqual` & `LessEqualPartly` :)

Comment (Thor-wl) [reply pkg/scheduler/api/resource_info.go:405]: > prefer to rename to `LessEqual` & `LessEqualPartly` :)

Sounds great! That will be brief and to the point. Let me update it.

Comment (Thor-wl) [reply pkg/scheduler/api/resource_info.go:374]: > prefer to rename to `LessEqual` & `LessEqualPartly` :)

Sounds great. That will be brief and to the point.

Comment (Thor-wl) [reply pkg/scheduler/api/resource_info.go]: Updated

**Review by wpeng102 — APPROVED** [review]
/lgtm

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**